client/src/containers/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import SearchBar from './search_bar';
+import ProductList from '../components/product-list'
+import Pagination from '../components/pagination'
+// import { Provider } from 'react-redux'
+import { connect } from 'react-redux.1'
```

Comment:
I'm a bit confused why you have a .1 tacked on to every reference to 'react-redux' in your project, because this is breaking your app. After removing every instance of 'react-redux.1' (changing it back to 'react-redux') the app runs fine. If this was intentional, I'd not recommend ever trying to change the name of an npm package.

Reply:
For what its worth, I just pushed a new commit fixing this issue. Not sure why the react-redux package didn't work on my end while working on your end. Assuming it's the result of some error I installing while installing and transferring between the client's node_modules file and the overall application's node_modules file. Taking the loss regardless.